web/src/settings_org.ts

```diff
@@ -517,21 +558,27 @@ export function discard_realm_property_element_changes(elem) {
         case "realm_can_delete_any_message_group":
         case "realm_can_delete_own_message_group":
         case "realm_can_manage_all_groups":
+            // TODO(evy): actually see what's possible here
+            assert(typeof property_value === "string" || typeof property_value === "number");
             settings_components.set_dropdown_list_widget_setting_value(
                 property_name,
                 property_value,
             );
             break;
         case "realm_default_language":
+            // TODO(evy): actually see what's possible here
+            assert(typeof property_value === "string");
             $("#org-notifications .language_selection_widget .language_selection_button span").attr(
                 "data-language-code",
                 property_value,
             );
             $("#org-notifications .language_selection_widget .language_selection_button span").text(
-                get_language_name(property_value),
+                get_language_name(property_value) ?? "", // TODO(evy) or assert not undefined?
```

Comment:
this shouldn't be undefined because it's coming from a preselected list of options